File: interface/src/main/java/com/kgc/pinyougou/service/ItemService.java
package com.kgc.pinyougou.service;

import com.kgc.pinyougou.domain.Item;
import com.kgc.pinyougou.utils.PageResult;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface ItemService {

	/**
	 * @Author Admin
	 * @description:  //TODO
	 * @param:
	 * @param pageNum   分页页数
	 * @param pageSize  分页记录数
	 * @return: com.kgc.pinyougou.utils.PageResult
	 * @time: 2020/4/25 22:11
	 */
	PageResult findAllItem(Integer pageNum, Integer pageSize, Item item);

	Integer insertItem(Item item);

	Item findOne(Long id);

	Integer updateItem(Item item);

	Integer deleteItem(Long [] ids);
}

File: pinyougou-manager-web/src/main/resources/static/js/service/itemService.js
//服务层
app.service('itemService',function($http){

    this.findAllItem=function (page,size,searchEntity) {
        return $http.post("../item/findAllItem?page="+page+"&size="+size,searchEntity)
    }

    this.findOne=function (id) {
        return $http.get("../item/findOne?id="+id)
    }

    this.del=function (ids) {
        return $http.post("../item/delItem",ids)
    }

    this.add=function (searchEntity) {
        return $http.post("../item/addItem",searchEntity)
    }

    this.update=function (searchEntity) {
        return $http.post("../item/updateItem",searchEntity)
    }
});
